openqaoa/backends/qpus/qaoa_pyquil_qpu.py

```diff
@@ -238,12 +238,19 @@ def get_counts(self, params: QAOAVariationalBaseParams) -> dict:
         params: QAOAVariationalBaseParams
             The QAOA parameters - an object of one of the parameter classes, containing 
             variable parameters.
+        n_shots: int
+            NOT WORKING YET. TODO: implement this.
+            The number of times to run the circuit. If None, n_shots is set to the default: self.n_shots
 
         Returns
         -------
         counts : dictionary
             A dictionary with the bitstring as the key and the number of counts as its value.
         """
+
+        # TODO: make n_shots useful
+        assert n_shots is None, "PyQuil does not support n_shots in get_counts"
```

Comment:
We should check how to solve this.

I don't know how to change the number of shots from this method.